src/main/java/org/javacord/bot/listeners/TalkToJamesListener.java

```diff
@@ -0,0 +1,50 @@
+package org.javacord.bot.listeners;
+
+import de.btobastian.sdcf4j.CommandHandler;
+import org.javacord.api.entity.Nameable;
+import org.javacord.api.entity.message.embed.EmbedBuilder;
+import org.javacord.api.event.message.MessageCreateEvent;
+import org.javacord.api.listener.message.MessageCreateListener;
+import org.javacord.api.util.logging.ExceptionLogger;
+import org.javacord.bot.Constants;
+
+import java.util.Arrays;
+
+public class TalkToJamesListener implements MessageCreateListener {
+
+    private final CommandHandler commandHandler;
+
+    /**
+     * Initializes the listener.
+     *
+     * @param commandHandler The command handler used to extract commands.
+     */
+    public TalkToJamesListener(CommandHandler commandHandler) {
+        this.commandHandler = commandHandler;
+    }
+
+    @Override
+    public void onMessageCreate(MessageCreateEvent event) {
+        if (!event.getServerTextChannel().map(Nameable::getName).orElse("").equals("talk-to-james")) {
```

Comment:
I'd suggest to use channel id like with the other we check.
It is defined in the `org.javacord.bot.Constants` anti-pattern. :-)
If you still want to go by name, at least use `event.getServerTextChannel().map(Nameable::getName).map("talk-to-james"::equals).orElse(FALSE)`, no need to compare empty string to expected name.